src/Controller/AbstractRestfulController.php

```diff
@@ -626,13 +615,8 @@ protected function jsonDecode($string)
             return json_decode($string, (bool) $this->jsonDecodeType);
         }
 
-        if (class_exists(Json::class)) {
-            return Json::decode($string, (int) $this->jsonDecodeType);
-        }
-
-        throw new DomainException(sprintf(
-            'Unable to parse JSON request, due to missing ext/json and/or %s',
-            Json::class
-        ));
+        throw new DomainException(
```

Comment:
Can we just drop the exception and reduce this method to `return json_decode(…`?

As you rightly mentioned, `ext-json` is available everywhere and you really have to foot gun yourself to disable it.

Reply:
Done